cmd/epp/main.go

```diff
@@ -23,6 +23,9 @@ import (
 )
 
 func main() {
+	// Register all known plugin factories
+	runner.RegisterAllPlgugins()
```

Comment:
Can plugin registration be moved to `plugins` package (under `scheduling` or global)?
It does not feel to be part of a `runner`.

Reply:
No. That will cause a loop in the imports. RegisterAllPlugins reference the actual plugin implementations, who in turn import the plugins package, especially for plugins.Plugin

Reply:
>I do think that the use of Object makes sense as it would natively separate the plugin type from its name, which is currently unclear, IMO. This is a side note - I'll open an issue to discuss to avoid sidetracking the feedback.

The problem with Object, is how to parse the Yaml. I don't think we want to centralize the parsing of the plugins' parameters. Only the plugin facxtory function needs to know how to parse its parameters.